_includes/selector.html

```diff
@@ -657,6 +658,9 @@
                     if (this.active_additional_packages.includes("TensorFlow")) {
                         notes = [...notes, "TensorFlow requires CUDA 12."];
                     }
+                    if (this.active_additional_packages.includes("NetworkX + nx-cugraph")) {
+                        notes = [...notes, "Selecting this option will install both standard NetworkX and a GPU-accelerated backend by RAPIDS"];
+                    }
```

Comment:
Any feedback on this messaging?
Preview:
![image](https://github.com/user-attachments/assets/d9b98e64-a20b-4568-8a45-51a6a9233657)